### PR TITLE
Clear Dictionary Activity

### PR DESCRIPTION
**Pull request by sebR18**
Lets try this again

### CODE REVIEW

File: UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx [hunk 1]
```diff
@@ -369,4 +369,60 @@
     <value>Dictionary value if exists</value>
     <comment>property description</comment>
   </data>
-</root>
+  <data name="Authentication_Category" xml:space="preserve">
```
Comment (k2zinger) [inline UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx:372]: remove multiple duplicates, leave the first entry earlier in the file

e.g.
lines 148-151 and 372-375, remove the section for 372-375 (duplicate)

**Review by k2zinger — CHANGES_REQUESTED** [review]
additional changes requested

File: .gitignore [hunk 2]
```diff
@@ -362,4 +362,5 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
\ No newline at end of file
+FodyWeavers.xsd
+/UiPathTeam.Extensions/UiPathTeam.Extensions.Activities.Design/Themes/Icons.xaml
```
[the remark this answers is not in the document]
Comment (k2zinger) [reply .gitignore:366]: ignoring this file means it is ignored from this point forward, so any valid changes to this file would be ignored
ignored files are typically files that are generated when user opens a project in Visual Studio / UiPath Studio and those files are not required to open the project on someone else's machine - this is especially important for temporary files

[the remark this answers is not in the document]
Comment (sebR18) [reply .gitignore]: removed the file from .gitignore

**Review by k2zinger — APPROVED** [review]

**Pull request merged**